AElf.Kernel.Tests/Managers/TransactionManagerTests.cs

```diff
@@ -1,17 +1,14 @@
 ﻿using System.Threading.Tasks;
 using AElf.Cryptography.ECDSA;
-//using AElf.Kernel.Managers;
-using Google.Protobuf;
 using Xunit;
 using AElf.Common;
 using AElf.Cryptography;
 using AElf.Kernel.Blockchain.Domain;
 using AElf.Kernel.TransactionPool;
 using Shouldly;
 
-namespace AElf.Kernel.Tests
+namespace AElf.Kernel.Tests.Managers
```

Comment:
Namespace is not correct.

Reply:
@zhxymh  For namespace format issue, new test projects namespace will be followed testing project namespace, others will update later since all existed test cases also have namespace format issue.